src/routes/solid-router/alternative-routers.mdx

```diff
@@ -0,0 +1,23 @@
+---
+title: Alternative Routers
+---
+
+## Hash Mode Router
+
+By default, Solid Router uses `location.pathname` as route path. You can simply switch to hash mode through the `source` property on `<Router>` component.
+
```

Comment:
I would like if we could avoid the use of `you` in this sentence and isntead just talk about how to switch to the Hash Mode Router and maybe why (if we haven't talked about it elsewhere).